commit c6509a24d6ba0f8560a81c032c30d169ba467b0a
Author: Masahiro Yamada <yamada.masahiro@socionext.com>
Date:   Tue Jun 4 19:13:58 2019 +0900

    kbuild: fix Kconfig prompt of CONFIG_HEADERS_CHECK
    
    Prior to commit 257edce66d31 ("kbuild: exploit parallel building for
    CONFIG_HEADERS_CHECK"), the sanity check of exported headers was done
    as a side-effect of build rule of vmlinux.
    
    That commit is good, but I missed to update the prompt of the Kconfig
    entry. For the sake of preciseness, lets' say "when building 'all'".
    
    Signed-off-by: Masahiro Yamada <yamada.masahiro@socionext.com>

diff --git a/lib/Kconfig.debug b/lib/Kconfig.debug
index cbdfae379896..7cdcb962358c 100644
--- a/lib/Kconfig.debug
+++ b/lib/Kconfig.debug
@@ -306,11 +306,11 @@ config DEBUG_FS
 	  If unsure, say N.
 
 config HEADERS_CHECK
-	bool "Run 'make headers_check' when building vmlinux"
+	bool "Run sanity checks on uapi headers when building 'all'"
 	depends on !UML
 	help
-	  This option will extract the user-visible kernel headers whenever
-	  building the kernel, and will run basic sanity checks on them to
+	  This option will extract the user-visible kernel headers when
+	  building the 'all' target, and will run basic sanity checks on them to
 	  ensure that exported files do not attempt to include files which
 	  were not exported, etc.